src/items/repositories/item.repository.ts

```diff
@@ -84,6 +84,18 @@ export class ItemRepository extends Repository<Item> {
       .orderBy('item.createdAt', 'DESC')
       .getOne();
   }
+  async listAnItem(itemId: string, price: number): Promise<Item> {
+    await this.update(
+      {
+        itemId,
+      },
+      { price: String(price), status: 1 }
```

Comment:
Please use the `ItemStatus` enum instead of hardcoded value.

Reply:
Go

Reply:
I didn't know